paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -309,6 +329,34 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
         }
     }
 
+    private fun setupNotes() {
```

Comment:
any way we can put this in the base sheet and not have to duplicate logic? I have the same concerns above where we might fix the stuck test issue in one place and be confused why it's not working in the other.